frontend/src/mocks/filterDataMock.ts

```diff
@@ -144,6 +144,8 @@ export const mockDetails = [
     classification: 'Core Operational',
     name: 'test name',
     description: 'test',
+    administrativeArea: 'Victoria',
+    assessed: 10000,
```

Comment:
again, not sure if this was the best way to do this. Also I don't actually know what the filterDataMock is for

Reply:
its just a test utility that contains mocked objects, prevents us from having to recreate these every time we create a new test that needs parcels/buildings etx.